### ISSUE TITLE
ERROR CONNECTING KEEPASS TO KEYANYWHERE

### ISSUE THREAD

**Issue by JACKJATIN** [Kyrodan/KeeAnywhere#355]
this issue is happening when i am trying to simply connect to my google drive account the webpage show response that the authentication is successful but the keepass show following error. Kindly Help me solve the Issue.
![Screenshot 2022-09-13 013247](https://user-images.githubusercontent.com/90373086/189746578-9de3cbdd-3db8-405c-b43e-624d4598f561.png)
![Screenshot 2022-09-13 013316](https://user-images.githubusercontent.com/90373086/189746590-93a88b09-42f1-483e-aaa2-0f8c1a60010c.png)

**Comment by JACKJATIN** [comment]
Is there any official support available like email address or something?

**Comment by Jackabomb** [comment]
No, you're in the right place. You'll have to wait for @Kyrodan or someone else more knowledgeable to take a look at it. I don't recognize that error. The error message mentions the token being expired, but you just did the authorization a second ago. Is your system date/time off? Have you tried with another provider?

**Comment by JACKJATIN** [comment]
It is working great with one drive as i set after this . This issue only with Gmail.
